core/src/test/java/org/elasticsearch/deps/joda/SimpleJodaTests.java

```diff
@@ -283,6 +314,13 @@ public void testForInvalidDatesInEpochMillis() {
         formatter.parser().parseDateTime(randomFrom("invalid date", "12345678901234"));
     }
 
+    public void testThatEpochParserIsPrinter() {
+        FormatDateTimeFormatter formatter = Joda.forPattern("epoch_millis");
+        assertThat(formatter.parser().isPrinter(), is(true));
+        assertThat(formatter.printer().isPrinter(), is(true));
+        assertThat(formatter.printer().print(1433144433), is("1433144433"));
+    }
```

Comment:
maybe we can also have a simple test that makes sure that calling print then parse is idempotent?